src/Console/RunCommand.php

```diff
@@ -13,17 +12,25 @@ protected function configure() {
         $this
             ->setName('phive:run')
             ->setDescription('@todo')
-            ->addArgument('arguments', InputArgument::OPTIONAL, '@todo', 'help')
             ->setHelp(<<<EOT
 @todo The <info>phive:run</info> command needs a description.</info>
 @example <info>phive:run install phpunit</info>
 EOT
-            );
+            )
+            ->ignoreValidationErrors();
     }
 
     protected function execute(InputInterface $input, OutputInterface $output) {
-        $output->writeln('[WIP]');
-        $phiveBinary = new PhiveBinary(__DIR__ . '/../../bin/phive.phar');
-        $output->writeln(sprintf('%s %s', $phiveBinary, $input->getArgument('arguments')));
+        $phiveBinary = new PhiveBinary;
+
+        if (false === $phiveBinary->exists()) {
+            return $output->writeln('@todo');
+        }
+
+        $argv = $_SERVER['argv'];
```

Comment:
I share your concerns. 

But I cannot use [$input](https://github.com/symfony/console/blob/fbbc77cf3f7fb45a79e6497f2dae99c42861f9ec/Input/ArgvInput.php) because of the required input definition needed in this class. I tried to configure the definition but it still seems impossible to passthrough the given CLI arguments to Phive. 

So I disabled the argument validation [here](https://github.com/phar-io/composer-plugin/blob/feature/first-working-version/src/Console/RunCommand.php#L20) and do the stuff in four lines.

I can only wrap this super global.